### ISSUE TITLE
The recent 1.4.0-phetio should be 1.5.0-phetio

### ISSUE THREAD

**Issue by phet-steele** [phetsims/build-an-atom#163]
The recently created test tasks, phetsims/tasks/issues/850 and phetsims/tasks/issues/847, should have been made from a 1.5-phetio branch. 1.4.0 was already delivered months ago: https://phet-io.colorado.edu/sims/build-an-atom/1.4.0-phetio/wrappers/index/.

Tagging @jbphet.

**Comment by zepumph** [comment]
Oops, not sure how that happened. Sorry about that. 
I turned the current RC shas into a branch called 1.5-phetio. The next release will be 1.5.0-phetiorc.1, let me know if that makes less sense than another option.

I updated the 1.4-phetio dependencies to reflect the latest 1.4-phetio release: https://phet-io.colorado.edu/sims/build-an-atom/1.4.1-phetio/dependencies.json in case for maintenance releases (I don't think we will need to, but just in case)
